man/journal-upload.conf.xml

```diff
@@ -60,6 +60,27 @@
         <xi:include href="version-info.xml" xpointer="v232"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Compression=</varname></term>
+
+        <listitem><para>Compression algorithm to be applied to logs data before sending.
+        Takes one of <literal>NONE</literal>, <literal>ZSTD</literal>, <literal>XZ</literal>, and <literal>LZ4</literal>.
```

Comment:
No no, please make this setting take lower case.

Reply:
actually what is a problem to have these values in upper case

Reply:
That's our usual syntax in .conf.
Please introduce tables one is upper case, and the other is lower. Then, use lower case table here.

Reply:
do you mean a separate table? what name do you want to use for it?

Reply:
added compression_lowercase_table